SpatialGDK/Source/SpatialGDK/Public/SpatialView/OpList/EntityComponentOpList.h

```diff
@@ -54,6 +54,8 @@ class EntityComponentOpListBuilder
 
 	OpList CreateOpList() &&;
 
+	EntityComponentOpListData& GetOpListData() { return *OpListData; }
```

Comment:
nit: this feels like it defeats the purpose of the builder. But at the same time I guess it hasn't been constructed yet. The only logic we need to expose is the remove if- could we instead expose a `RemoveAllUpdates` on the builder which takes the same predicate, so we don't enable other ways of modifying the op list as it's constructed?

Reply:
Actually I was surprised to see it in the GDK, I assumed it was mostly a test facility.
I'd rather pop it open that way and leave the convoluted logic we need for the dispatcher in the dispatcher rather than adding edition methods.
Maybe I could just replace it with the OpListData themselves and leave the builder alone.

Reply:
spoke offline, a builder probably isn't the right abstraction for this. Replacing with oplistdata is likely a better option, but as this is a bugfix depends on how long it would take to do